client/src/ui/BitbakeTerminal.ts

```diff
@@ -175,7 +175,7 @@ export class BitbakePseudoTerminal implements vscode.Pseudoterminal {
   }
 }
 
-class BitbakeTerminal {
+export class BitbakeTerminal {
```

Comment:
Not sure we want to break the convention here. You should use the existing exported functions to use bitbake terminals.

Reply:
The existing functions are deeply tied with each other which are specifically for bitbake commands. I just needed to replicate the ones that I need here.

Reply:
I could try to export `runBitbakeTerminalScript` instead of the class `BitbakeTerminal` which should avoid the code duplication you mentioned below.